tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```diff
@@ -1035,6 +1042,235 @@ def _(
     return x.new_empty((b, d), dtype=o_dtype)
 
 
+def pad_up(x: int, y: int) -> int:
+    return ((x + y - 1) // y) * y
+
+
+class CuteDSLNVFP4BlackwellLinear(TunableRunner):
+    kernel_dict = dict()
+
+    tuning_config = TuningConfig(
+        dynamic_tensor_specs=(DynamicTensorSpec(
+            0, 0, get_last_power_of_2_num_tokens_buckets,
+            last_positive_power_of_2), ),
+        constraint_specs=(ConstraintSpec(2, 0, fp4_scale_infer_shape), ),
+    )
+
+    def __init__(self):
+        super().__init__()
+
+    def get_valid_tactics(
+        self,
+        inputs: List[torch.Tensor],
+        profile: OptimizationProfile,
+        **kwargs,
+    ) -> List[int]:
+        m = inputs[0].shape[0]
+        n = inputs[1].shape[0]
+        k = inputs[0].shape[1]
+        # Note: the input tensor use uint8 to store fp4, so the real_k is k * 2
+        real_k = k * 2
+        batch_size = 1
+        # m,k
+        a_major = "k"
+        # n, k
+        b_major = "k"
+        # m, n
+        c_major = "n"
+        sf_vec_size = 16
+
+        # full shamoo
+        mma_tiler_mn_candi = [(256, 128), (128, 128), (128, 256), (256, 256),
```

Comment:
A nit.

Normally we use full name like `candidates`, do not use half words like `candi`.